docs/introduction.md

```diff
@@ -57,12 +57,14 @@ The following functions are provided:
 - Arithmetic, comparison, shifting, and validation operations for packed decimal data.
 - Conversion operations between different binary coded decimal and Java binary types.
 - Marshalling operations: marshalling and unmarshalling Java binary types, such as `short`, `int`, `long`, `float`, and `double`, to and from byte arrays.
+- External decimals support with all four sign configurations: sign embedded trailing, sign embedded leading, sign separate trailing, and sign separate leading. It also accommodates sign embedded trailing with spaces.
```

Comment:
Same as before, this might be more appropriate here
```
- Validation operations: validation of packed and external decimals. All four sign configurations for External decimals are supported: sign embedded trailing, sign embedded leading, sign separate trailing, and sign separate leading. It also accommodates sign embedded trailing with spaces.
```

Reply:
Based on https://github.com/eclipse-openj9/openj9-docs/pull/1506#discussion_r2053290417, this is no longer needed